diff --git a/src/Plugin/views/row/Calendar.php b/src/Plugin/views/row/Calendar.php
index f04727c2..fb6ad1e6 100644
--- a/src/Plugin/views/row/Calendar.php
+++ b/src/Plugin/views/row/Calendar.php
@@ -10,6 +10,7 @@
 use Drupal\taxonomy\Entity\Term;
 use Drupal\views\Plugin\views\argument\Date;
 use Drupal\Core\Datetime\DateFormatter;
+use Drupal\Core\Datetime\DrupalDateTime;
 use Drupal\Core\Entity\Entity;
 use Drupal\Core\Form\FormStateInterface;
 use Drupal\views\Plugin\views\display\DisplayPluginBase;
@@ -433,8 +434,8 @@ public function render($row) {
 //        $db_tz   = date_get_timezone_db($tz_handling, isset($item->$tz_field) ? $item->$tz_field : timezone_name_get($dateInfo->getTimezone()));
 //        $to_zone = date_get_timezone($tz_handling, isset($item->$tz_field)) ? $item->$tz_field : timezone_name_get($dateInfo->getTimezone());
 
-        $item_start_date = \DateTime::createFromFormat($storage_format, $row->{$info['query_name']});
-        $item_end_date = \DateTime::createFromFormat($storage_format, $row->{$info['query_name']});
+        $item_start_date = DrupalDateTime::createFromFormat($storage_format, $row->{$info['query_name']}, DATETIME_STORAGE_TIMEZONE);
+        $item_end_date = DrupalDateTime::createFromFormat($storage_format, $row->{$info['query_name']}, DATETIME_STORAGE_TIMEZONE);
 
         // @todo don't hardcode
 //        $granularity = date_granularity_precision($cck_field['settings']['granularity']);
