diff --git a/calendar.views.inc b/calendar.views.inc
index 43a1e28..d54da01 100644
--- a/calendar.views.inc
+++ b/calendar.views.inc
@@ -63,7 +63,7 @@ function _calendar_create_field_arguments($field_id, $group = '') {
 /**
  * Implements hook_field_views_data_alter().
  */
-function calendar_datetime_field_views_data_alter(array &$data, FieldStorageConfigInterface $field_storage) {
+function calendar_field_views_data_alter(array &$data, FieldStorageConfigInterface $field_storage) {
   foreach ($data as $table_name => $table_data) {
     // Set the 'datetime' filter type.
     if (isset($data[$table_name][$field_storage->getName() . '_value'])
diff --git a/src/Plugin/views/argument/DatetimeYearWeekDate.php b/src/Plugin/views/argument/DatetimeYearWeekDate.php
index 72905d8..5437fb5 100644
--- a/src/Plugin/views/argument/DatetimeYearWeekDate.php
+++ b/src/Plugin/views/argument/DatetimeYearWeekDate.php
@@ -2,14 +2,14 @@
 
 namespace Drupal\calendar\Plugin\views\argument;
 
-use Drupal\datetime\Plugin\views\argument\Date as DateTimeDate;
+use Drupal\datetime\Plugin\views\argument\Date;
 
 /**
  * Argument handler for a day.
  *
  * @ViewsArgument("datetime_year_week")
  */
-class DatetimeYearWeekDate extends DateTimeDate{
+class DatetimeYearWeekDate extends Date {
 
   /**
    * {@inheritdoc}
diff --git a/src/Plugin/views/argument/TimeStampYearWeekDate.php b/src/Plugin/views/argument/TimeStampYearWeekDate.php
index 906c10e..8608eba 100644
--- a/src/Plugin/views/argument/TimeStampYearWeekDate.php
+++ b/src/Plugin/views/argument/TimeStampYearWeekDate.php
@@ -2,15 +2,14 @@
 
 namespace Drupal\calendar\Plugin\views\argument;
 
-
-use Drupal\views\Plugin\views\argument\Date as NumericDate;
+use Drupal\views\Plugin\views\argument\Date;
 
 /**
  * Argument handler for a day.
  *
  * @ViewsArgument("date_year_week")
  */
-class TimeStampYearWeekDate extends NumericDate{
+class TimeStampYearWeekDate extends Date {
 
   /**
    * {@inheritdoc}
