diff --git a/calendar/calendar.module b/calendar/calendar.module
index 61a17a0..713fd95 100644
--- a/calendar/calendar.module
+++ b/calendar/calendar.module
@@ -272,32 +272,37 @@ function calendar_views_query_alter($view, $query) {
     // Loop through statements in this query.
     foreach ($query->where as $statement_key => $statement) {
 
-      // Loop through conditions within this statement.
-      foreach ($statement['conditions'] as $condition_key => $condition) {
+      // If view style plugin is calendar...
+      if ($view->style_plugin->getPluginId() == 'calendar') {
 
-        // If this condition is a formula...
-        if ($condition['operator'] == 'formula') {
+        // Loop through conditions within this statement.
+        foreach ($statement['conditions'] as $condition_key => $condition) {
 
-          // If this condition formula uses the date format function...
-          if (strpos($condition['field'], 'DATE_FORMAT') !== false) {
+          // If this condition is a formula...
+          if ($condition['operator'] == 'formula') {
 
-            // Retrieve this condition's current field value for modification.
-            $condition_field = $query->where[$statement_key]['conditions'][$condition_key]['field'];
+            // If this condition formula uses the date format function...
+            if (strpos($condition['field'], 'DATE_FORMAT') !== false) {
 
-            // Split condition field into parts.
-            $condition_field_parts = array_map('trim', explode('=', $condition_field));
+              // Retrieve this condition's current field value for modification.
+              $condition_field = $query->where[$statement_key]['conditions'][$condition_key]['field'];
 
-            // If the split returned 2 parts, as expected...
-            if (count($condition_field_parts) == 2) {
+              // Split condition field into parts.
+              $condition_field_parts = array_map('trim', explode('=', $condition_field));
 
-              // Modify query to include records from the next month and year.
-              $condition_field_parts[1] = '(' . $condition_field_parts[1] . ',' . $condition_field_parts[1] . ' + 1, ' . $condition_field_parts[1] . ' + 89)';
-              $condition_field = implode(' IN ', $condition_field_parts);
+              // If the split returned 2 parts, as expected...
+              if (count($condition_field_parts) == 2) {
 
-            }
+                // Modify query to include records from the next month and year.
+                $condition_field_parts[1] = '(' . $condition_field_parts[1] . ',' . $condition_field_parts[1] . ' + 1, ' . $condition_field_parts[1] . ' + 89)';
+                $condition_field = implode(' IN ', $condition_field_parts);
+
+              }
 
-            // Substitute the original field value for the modified one.
-            $query->where[$statement_key]['conditions'][$condition_key]['field'] = $condition_field;
+              // Substitute the original field value for the modified one.
+              $query->where[$statement_key]['conditions'][$condition_key]['field'] = $condition_field;
+
+            }
 
           }
 
